darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -95,7 +95,8 @@ def __init__(self,
                  log_tensorboard: bool = False,
                  nr_epochs_val_period: int = 10,
                  torch_device_str: Optional[str] = None,
-                 force_reset=False):
+                 force_reset=False,
+                 save_checkpoints=True):
```

Comment:
I'm actually thinking that we should perhaps default this one to `False`, wdyt? I think it can cause issues for some users because it's happening silently, can generate files quite large, and even though we keep only the last 5, each model name is (by default) unique so each model instance will generate 5 files by default.

Reply:
agreed!

Reply:
```suggestion
                 save_checkpoints=False):
```